conda-recipe/meta.yaml

```diff
@@ -29,20 +29,17 @@ requirements:
     - humanize
     - numpy
     - pytz
+    - boltons
 
 test:
   requires:
     - nslsii_dev_configuration
 
-  # Python imports
   imports:
+    # Import all the packages
     - metadatastore
-    - metadatastore.commands
```

Comment:
This is one reason to switch to using `conda build` on travis (assuming we can find a way to cope with the deps).  

I am in favor of 1 and shaming people that forget to update meta.yaml